src/main/java/net/minecraftforge/event/ForgeEventFactory.java

```diff
@@ -686,11 +686,11 @@ public static boolean saplingGrowTree(IWorld world, Random rand, BlockPos pos)
         return event.getResult() != Result.DENY;
     }
 
-    public static void fireChunkWatch(boolean watch, ServerPlayerEntity entity, ChunkPos chunkpos, ServerWorld world)
+    public static void fireChunkWatch(boolean wasLoaded, boolean load, ServerPlayerEntity entity, ChunkPos chunkpos, ServerWorld world)
```

Comment:
This is more a question for the forge team than you directly, but does this need some sort of fallback given technically this is a "breaking change", and now that forge has an RB.

Reply:
Normally the answer would be yes, however this is an event factory used within the chunk manager. No mod should be calling this. Lex has the last word as always, but this doesn't feel like it conflicts with the "no breaking changes" rule.

Reply:
Yes it falls under binary compatibility